Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/BoundingBox/BoundingBox.cs

```diff
@@ -821,18 +822,20 @@ public bool ProximityEffectActive
         [SerializeField]
         [Tooltip("How far away should the hand be from a handle before it starts scaling the handle?")]
         [Range(0.005f, 0.2f)]
+        private float handleMediumProximity = 0.1f;
         /// <summary>
         /// Distance between handle and hand before proximity scaling will be triggered.
         /// </summary>
-        private float handleMediumProximity = 0.1f;
+        public float HandleMediumProximity => handleMediumProximity;
```

Comment:
should this also have a setter?

Reply:
i think in case of bounding box we don't need it as it's going to be deprecated soon - however i need the get for the upcoming migration to read out the values

Reply:
We expect `BoundingBox` to hang around for a few versions after we depreciate right? Maybe it's worth having the setters in case people are still using it?

Reply:
i can add them, but they would only need them in case they want to code configure existing bounding boxes. I'd rather not add anything because I don't want to invest time to write up tests for those runtime config changes - if someone requests this I can do it, but i wouldn't just for the sake of having setters there. The getters are also only there because i  need them for migration :)